ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/generator/ChainBuilder.java

```diff
@@ -130,6 +141,10 @@ public Optional<BlobsSidecar> getBlobsSidecar(final Bytes32 blockRoot) {
     return Optional.ofNullable(blobsSidecarsByHash.get(blockRoot));
   }
 
+  public Optional<BlobSidecar> getBlobSidecar(final Bytes32 blockRoot) {
+    return Optional.ofNullable(blobSidecarsByHash.get(blockRoot));
```

Comment:
The could be more than one in every block

Reply:
Done. The `ChainBuilder` could be refactored further when adding integration tests later on